test/powershell/Modules/Microsoft.PowerShell.Utility/assets/TestImportCsvQuoteDelimiter_EmptyValue.csv

```diff
@@ -0,0 +1,2 @@
+a1""a3
```

Comment:
Would prefer to generate these at runtime than have it checked in as it's easier to see the test file in the test code than open another file.  Something like:

```powershell
$emptyValueCsv = @'
a1""a3
v1"v2"v3
'@

Set-Content testdrive:/EmptyValue.csv -Value $emptyValueCsv
```

Reply:
I looked into using a here-string earlier, but the lack of indentation for the string terminator made the code look quite messy.

I couldn't quite find an elegant way to add the here-string, but I've gone an added one for the different file contents.